action.yml

```diff
@@ -176,6 +180,12 @@ runs:
             $args += @("--enable-plugin=$plugin");
           }
         }
+        if ("${{ inputs.include-qt-plugins }}" -ne ''){
```

Comment:
This is bound to create an options usage that will not work for Nuitka 1.6 yet, but for 1.7 and current develop it will do the right thing, even if the input is a list, it will no longer override, but amend values to the plugin.